vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="tw-mt-[15rem] tw-mb-[30rem]">
+    <div class="tw-container tw-flex tw-flex-col tw-items-center">
+      <div class="tw-max-w-[17.5rem] md:tw-max-w-[41.125rem]">
+        <div
+          class="tw-text-center tw-font-roboto-bold tw-text-black-core/[.87] tw-text-[1.875rem] md:tw-text-[2.625rem] xl:tw-text-[3.438rem] tw-leading-[2.5rem] md:tw-leading-[2.75rem] xl:tw-leading-[5rem]"
+        >
+          Do we have a team with the right skills?
+        </div>
+      </div>
+      <div
+        class="tw-mt-[1.125rem] tw-max-w-[23.313rem] md:tw-max-w-[50.688rem]"
+      >
+        <div
+          class="tw-w-full tw-text-center tw-font-inter-medium tw-text-black-core/[0.6] tw-text-[1rem] md:tw-text-[1.25rem] xl:tw-text-[1.5rem] tw-leading-[1.375rem] md:tw-leading-[2rem] xl:tw-leading-[2.625rem]"
+        >
+          Well, at least the community says Hell Yeah. Our blogs hosted on
+          medium have
+          <span class="v2-canopas-gradient-text tw-font-inter-bold">100k+</span>
+          minutes monthly reading time and it’s only rising.
+        </div>
+      </div>
+    </div>
+    <div
+      class="tw-relative tw-flex tw-flex-col tw-items-center md:tw-items-start tw-mt-[6.813rem]"
+    >
+      <img
+        v-if="width < 680"
+        :src="bgMobile"
+        loading="lazy"
+        :class="blogs.length == 2 ? '-tw-top-[2rem]' : '-tw-top-[3rem]'"
+        class="tw-absolute tw-right-0"
+        alt="blog-background-image"
+      />
+      <img
+        v-else
+        :src="bg"
+        :class="blogs.length == 2 ? '-tw-top-[2rem]' : 'tw-top-0'"
+        class="tw-absolute tw-right-0"
+        alt="blog-background-image"
+      />
+
+      <div class="md:tw-mt-[4.625rem] tw-z-[1]">
+        <hr
+          class="tw-hidden md:tw-block tw-mt-[1.125rem] md:tw-ml-[4.375rem] lg:tw-ml-[8.25rem] 2xl:tw-ml-[9.313rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063] tw-bg-[#C6C6C6]"
+        />
+        <div
+          v-for="(blog, index) in blogs"
+          :key="blog"
+          @mouseover="activeIndex = index"
+          @mouseleave="activeIndex = index"
```

Comment:
On `over` and `leave` both, we will do the same?

Reply:
we have to hold the current index when user leaves the container also right that's why done this